media-libs/jbigkit/patches/jbigkit-2.1.patchset

```diff
@@ -0,0 +1,80 @@
+From 267d0811689d2115b42dce345c00be5cd2ccf86d Mon Sep 17 00:00:00 2001
+From: begasus <begasus@gmail.com>
+Date: Sat, 7 Apr 2018 17:31:24 +0200
+Subject: remove -Wno-unused-result, add debug support
+
+
+diff --git a/Makefile b/Makefile
+index 9f7e3ba..645f1c5 100644
+--- a/Makefile
++++ b/Makefile
+@@ -4,8 +4,8 @@
+ CC = gcc
+ 
+ # Options for the compiler: A high optimization level is suggested
+-CFLAGS = -O2 -W -Wno-unused-result
+-# CFLAGS = -O -g -W -Wall -Wno-unused-result -ansi -pedantic # -DDEBUG
++# CFLAGS = -O2 -W -Wno-unused-result
++CFLAGS = -O -g -W -ansi -pedantic # -DDEBUG
+ 
+ export CC CFLAGS
+ 
+-- 
+2.16.2
+
+
+From 6b76dac0aaa996bdb546ccc51f113ec8bcc719d6 Mon Sep 17 00:00:00 2001
```

Comment:
Since you are the author of this patchset and since it is not yet in master, I think it would be nice to squash the three commits in it.

BTW, if you follow the suggestion I made in point 3) of [my previous message](https://github.com/haikuports/haikuports/pull/2376#pullrequestreview-110255091), then this will fix the issue spotted in point 1) (of the same message.)
Reminder: if you wish to use the `-g` in the `CFLAGS` then you should drop the `-c` in `CFLAGS=-fPIC -g -c -Wall`. (That's because you can't pass both `-c` and `-o` in the same call.)

Another option is to not use the `-g`, but in that case it would be better to not create the debuginfo package.